pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -133,13 +133,15 @@ const (
 
 // predicateOrdering is the ordering of predicate execution.
 var predicateOrdering = []string{
+	GeneralPred,
 	CheckNodeUnschedulablePred,
-	GeneralPred, HostNamePred, PodFitsHostPortsPred,
-	MatchNodeSelectorPred, PodFitsResourcesPred, NoDiskConflictPred,
-	PodToleratesNodeTaintsPred, CheckNodeLabelPresencePred,
-	CheckServiceAffinityPred, MaxEBSVolumeCountPred, MaxGCEPDVolumeCountPred, MaxCSIVolumeCountPred,
-	MaxAzureDiskVolumeCountPred, MaxCinderVolumeCountPred, CheckVolumeBindingPred, NoVolumeZoneConflictPred,
+	HostNamePred, PodToleratesNodeTaintsPred,
+	MatchNodeSelectorPred, CheckServiceAffinityPred, PodFitsHostPortsPred,
+	PodFitsResourcesPred, NoDiskConflictPred, MaxCinderVolumeCountPred,
+	MaxEBSVolumeCountPred, MaxGCEPDVolumeCountPred, MaxCSIVolumeCountPred, MaxAzureDiskVolumeCountPred,
+	CheckVolumeBindingPred, NoVolumeZoneConflictPred,
 	EvenPodsSpreadPred, MatchInterPodAffinityPred,
+	CheckNodeLabelPresencePred,
```

Comment:
I add comment at here https://github.com/kubernetes/kubernetes/pull/95849#issuecomment-720465309

I doesn't feel this good way to sort. Maybe we can try to re-sort plugins order after apply the policy